src/property/FaceliftProperty.h

```diff
@@ -77,6 +85,19 @@ class PropertyBase
         return m_ownerSignal;
     }
 
+    bool &readiness()
```

Comment:
I feel like isReady() would be a better name

Reply:
and it should be const

Reply:
That might be rather confusing, if in all other places (even as annotation) we use "readiness" and only here "isReady". I tend to leave it as it is.

A non-const reference is returned here, so the function cannot be made const.